src/components/pets/pets.css

```diff
@@ -56,3 +56,48 @@
     5px 5px 15px 5px rgba(0, 0, 0, 0);
   box-shadow: 0px 10px 13px -7px #000000, 5px 5px 15px 5px rgba(0, 0, 0, 0);
 }
+
+.image {
+  position: relative;
+  width: 400px;
+}
+
+.image__img {
+  display: block;
+  width: 100%;
+}
+
+.image__overlay {
```

Comment:
I think the overlay itself is a touch too dark. I'd lighten it a tad.

Can you add rounded corners too? u could use `border-radius`

Reply:
Border-radius of 15% added